sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -504,6 +509,7 @@ func resourceSumologicMonitorsLibraryMonitorRead(d *schema.ResourceData, meta in
 	d.Set("created_by", monitor.CreatedBy)
 	d.Set("created_at", monitor.CreatedAt)
 	d.Set("monitor_type", monitor.MonitorType)
+	d.Set("evaluation_delay", monitor.EvaluationDelay)
```

Comment:
you might want to do something similar to what we do for timerange by calling `PositiveTimeRange` function

Reply:
For this field, "0s" is allowed - that is not positive correct?

Reply:
In this context, positive just means without the negative sign (if you look at the implementation of PositiveTimeRange function you can see it just strips the leading "-" minus sign, if it exists)